### DISCUSSION TITLE
v0.72.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#5919] [Announcements]
### Features and improvements

* Local file system storage is now available as an alternative to Minio/S3. For more details on setup – [documentation here](https://docs.getoutline.com/s/hosting/doc/file-storage-N4M0T6Ypu7)
* Documents now include an emoji picker in https://github.com/outline/outline/pull/4338
* Mermaid diagram rendering performance improved by @antran22 in https://github.com/outline/outline/pull/5852
* Embedding videos is now possible in https://github.com/outline/outline/pull/5866
* Emails sent from Outline now include actions which will be displayed in compatible email clients
* Nested docs more than one level deep in shared sidebar are now collapsed by default in https://github.com/outline/outline/pull/5208
* Base64 encoded and remote hosted images are now transferred to storage when creating a document in https://github.com/outline/outline/pull/5740
* Sidebar toggle moved into the sidebar itself instead of overlaying document in https://github.com/outline/outline/pull/5749
* "Inter" is now the default typeface on Windows and Linux in https://github.com/outline/outline/pull/5741
* Add document unsubscribe link in email footer in https://github.com/outline/outline/pull/5762
* Team members can now choose their own setting for whether "Edit" button is enabled in https://github.com/outline/outline/pull/5772
* Exporting an individual document now bundles attachments in a zip file in https://github.com/outline/outline/pull/5778
* Templates management moved to settings in https://github.com/outline/outline/pull/5811
* It is now possible to delete imports and all their associated data in https://github.com/outline/outline/pull/5907
* "Full width" setting is now respected when creating a document from a template
* C++ syntax highlighting added
* [Pitch](https://pitch.com/) embeds are now supported
* [Valtown](https://www.val.town/) embeds are now supported
* [Tldraw](https://tldraw.com/) embeds are now working again
* New and updated translations
* 🇺🇦 Ukranian language support added

### Fixes

* Re-position hover preview correctly to prevent going out of page bounds by @apoorv-mishra in https://github.com/outline/outline/pull/5702
* Fixed document sidebar animation in Safari by @lampy255 in https://github.com/outline/outline/pull/5766
* Fixed initials not display on notification avatars in https://github.com/outline/outline/pull/5803
* Fixed an issue where the app would switch back to default installation language when navigating to root url
* A revision is now always written when leaving "Edit" mode with changes
* Correctly resize full width images when table of contents is opened/closed in https://github.com/outline/outline/pull/5826
* Upload placeholders are no longer removed when another user edits the document in https://github.com/outline/outline/pull/5898
* Fixed an issue where attachment placeholders could get stuck in the document in a "Loading…" state
* Fixed an issue where import from JSON could not be completed more than once
* Improved toolbar display on mobile
* Previous path now remembered after forced logout #5893
* Fixed an issue where mobile styling would be used for toolbar on Windows touchscreen laptops
* Fixed @mention menu hanging after backspace
* Removed "0" count on drafts in the sidebar
* Improved client-side caching headers on attachments and images
* Improved performance of rendering context menus
* Fixed flickering on Mermaid diagrams while document loads
* Fixed uncaught error that could leave data exports in "Processing" state
* Emoji and @mention menus now work in headings
* Pasting a single line of code into a document now uses inline code formatting rather than a block by default

### API

* `published` and `includeChildDocuments` properties now available on `shares.create`
* `createdAt` and `emoji` properties now available on `documents.create`
* comments API endpoints now check that the feature is enabled

## New Contributors
* @antran22 made their first contribution in https://github.com/outline/outline/pull/5852

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.72.0'>v0.72.0</a>.</em>

**Comment by Limezy** [comment]
Many thanks for that great update. Having Outline fully working without MinIO is a real pleasure and piece of mind !

**Comment by almereyda** [comment]
❤️ for Inter #5741 and the unsubscribe link #5762

It's never been that Outline has just released a new version, when I had scheduled the next upgrade, such as right now.

**Comment by alkanna** [comment]
Thanks for the hard work !
We are now starting migration from ClickUp docs to Outline, we got it working on our kubernetes cluster using :
- Google Cloud Storage
- Cloud SQL (Postgres)
- Behind Kong API Gateway

So far so good 👍

**Comment by prologic** [comment]
Question on this feature:

> Local file system storage is now available as an alternative to Minio/S3. For more details on setup – [documentation here](https://docs.getoutline.com/s/hosting/doc/file-storage-N4M0T6Ypu7)

How would you go about migrating off of S3/MinIO to local storage? Can we just `mc mirror` the contents of the Bucket to a local filesystem? 🤔

**Reply by Limezy** [reply]
Yep that's what I did. You just need to create the /var/lib/data folder. I have the impression it's hardcoded and you can't change but I may be wrong.

**Reply by Limezy** [reply]
To be precise I did a mirror --preserve

**Comment by nh4ttruong** [comment]
It seems to you not updated `latest` tag on `outlinewiki/outline` image for v0.72.0 🥲

**Reply by tommoor** [reply]
`latest` tag was added 👍

**Comment by Karmanoid** [comment]
many thanks for update. Tags were expected in this update, Am I wrong?

**Comment by bmm-alc** [comment]
Hi @tommoor 

Thanks to all contributors for the work on outline.
Especially fixing the bug where the sidebar was folding when moving a document.

Best

**Comment by Anbool** [comment]
Nice!Thank you!

**Comment by inkloop** [comment]
Great work! Special Thank You for the support of 🇺🇦 Ukrainian language.

**Comment by bmm-alc** [comment]
thanks for the quick .1 release with fixes
